packager/packager.cc

```diff
@@ -2218,19 +2225,19 @@ void Packager::dumpPackageInfo(const core::GlobalState &gs, std::string outputFi
         writer.StartObject();
 
         writer.String("name");
-        writer.String(absl::StrJoin(pkg.fullName(), "::", core::packages::NameFormatter(gs)));
+        writer.String(pkg.mangledName().mangledName.show(gs));
```

Comment:
This `dumpPackageInfo` function is only used by our test harness.

The only thing that matters is that the `"name"` be in the same format as the `"import"` and `"testImport"` lists, because it will index into some unordered maps with those things as string keys.

Reply:
(Stripe's codebase does not use `--dump-package-info`, so the format does not matter.)